app/main/services/search_service.py

```diff
@@ -3,13 +3,18 @@
 from elasticsearch import Elasticsearch, TransportError
 from app.mapping import SERVICES_MAPPING
 from app.main.services.response_formatters import \
-    convert_es_status, convert_es_results
+    convert_es_status, convert_es_results, generate_pagination_links
 from app.main.services.query_builder import construct_query
+from flask import current_app, url_for
 
 es_url = os.getenv('DM_ELASTICSEARCH_URL')
 es = Elasticsearch(es_url)
 
 
+def refresh(index_name):
+    return es.indices.refresh(index_name)
+
+
 def response(status_code, message):
     return {"status_code": status_code, "message": message}
```

Comment:
Probably worthwhile - I used this only in tests, but bringing it in line should be done.